src/Components/Contact/Contact.js

```diff
@@ -7,58 +8,92 @@ import {
 } from "react-icons/ai";
 
 const Contact = () => {
+  const form = useRef();
+
+  const sendEmail = (e) => {
+    e.preventDefault();
+
+    emailjs
+      .sendForm(
+        "service_t5pviue",
+        "template_fr9k6ni",
+        form.current,
+        "user_xugJxJDFF667rtnlDTI02"
```

Comment:
if any of this is your private API token it's a best idea to move it to a private environment file otherwise it might be stolen